CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Changelog
 
-## Unreleased
+## 97.1.1
```

Comment:
**suggestion:** semver

Is this a patch? It's extending the API, so might be worth calling it a minor release?